TCAT/Controllers/RouteDetail+ContentViewController.swift

```diff
@@ -674,6 +670,26 @@ class RouteDetailContentViewController: UIViewController, GMSMapViewDelegate, CL
 
     }
 
+    func centerMap(on direction: Direction, overviewOfPath: Bool = false, drawerPreviewing: Bool = false) {
```

Comment:
prefer `Bool` variables begin with `is`, it's unclear what `overviewOfPath` represents here